neqo-transport/src/lib.rs

```diff
@@ -59,7 +59,7 @@ const ERROR_APPLICATION_CLOSE: TransportError = 12;
 const ERROR_AEAD_LIMIT_REACHED: TransportError = 15;
 
 #[derive(Clone, Debug, PartialEq, PartialOrd, Ord, Eq)]
-#[allow(clippy::pub_enum_variant_names)]
+// #[allow(clippy::pub_enum_variant_names)]
```

Comment:
you want to remove this?

Reply:
Yeah, for reasons I don't understand, this doesn't generate warnings.  But including the override fails in 1.54.0.  🤷‍♀️